pkg/storkctl/snapshot_test.go

```diff
@@ -95,5 +102,164 @@ func TestGetVolumeSnapshotsOneSnapshot(t *testing.T) {
+			require.Equal(t, expected, e)
+		})
+		testSnapshotsCommon(t, newCreateCommand, cmdArgs, &snapshots, expected, true)
+	}()
+}
+
+func TestCreateSnapshotsNoPVCName(t *testing.T) {
+	cmdArgs := []string{"volumesnapshots", "-p", "", "snap1"}
+
+	var snapshots snapv1.VolumeSnapshotList
+	expected := "error: PVC name needs to be given"
+	calledFatal := false
+
+	func() {
+		defer func() {
+			cmdutil.DefaultBehaviorOnFatal()
+		}()
+		cmdutil.BehaviorOnFatal(func(e string, code int) {
+			if calledFatal {
+				return
+			}
+			calledFatal = true
+			require.Equal(t, 1, code, "Unexpected error code")
+			require.Equal(t, expected, e)
+		})
+		testSnapshotsCommon(t, newCreateCommand, cmdArgs, &snapshots, expected, true)
+	}()
+}
+
+func TestCreateSnapshotsNoNamespace(t *testing.T) {
+	cmdArgs := []string{"volumesnapshots", "-p", "pvc_name", "snap1"}
+
+	var snapshots snapv1.VolumeSnapshotList
+	expected := "error: an empty namespace may not be set during creation"
+	calledFatal := false
+
+	func() {
+		defer func() {
+			// Restore cmdutil behavior.
+			cmdutil.DefaultBehaviorOnFatal()
+		}()
+		cmdutil.BehaviorOnFatal(func(e string, code int) {
+			if calledFatal {
+				return
+			}
+			calledFatal = true
+			require.Equal(t, 1, code, "Unexpected error code")
+			require.Equal(t, expected, e)
+		})
+		testSnapshotsCommon(t, newCreateCommand, cmdArgs, &snapshots, expected, true)
+	}()
+}
+
+func TestCreateSnapshots(t *testing.T) {
+	cmdArgs := []string{"volumesnapshots", "-p", "pvc_name", "snap1"}
+
+	var snapshots snapv1.VolumeSnapshotList
+	expected := `Snapshot snap1 created successfully
+
+`
+	testSnapshotsCommon(t, newCreateCommand, cmdArgs, &snapshots, expected, false)
+}
+
+func TestDeleteSnapshotsNoSnapshotName(t *testing.T) {
+	cmdArgs := []string{"volumesnapshots"}
+
+	var snapshots snapv1.VolumeSnapshotList
+	expected := "error: At least one argument needs to be provided for snapshot name"
+	calledFatal := false
+
+	func() {
+		defer func() {
+			cmdutil.DefaultBehaviorOnFatal()
+		}()
+		cmdutil.BehaviorOnFatal(func(e string, code int) {
+			if calledFatal {
+				return
+			}
+			calledFatal = true
+			require.Equal(t, 1, code, "Unexpected error code")
+			require.Equal(t, expected, e)
+		})
+		testSnapshotsCommon(t, newDeleteCommand, cmdArgs, &snapshots, expected, true)
+	}()
+}
+
+func TestDeleteSnapshotsNoPVCName(t *testing.T) {
+	cmdArgs := []string{"volumesnapshots", "snap1"}
+
+	var snapshots snapv1.VolumeSnapshotList
+	expected := "error: an empty namespace may not be set when a resource name is provided"
+	calledFatal := false
+
+	func() {
+		defer func() {
+			cmdutil.DefaultBehaviorOnFatal()
+		}()
+		cmdutil.BehaviorOnFatal(func(e string, code int) {
+			if calledFatal {
+				return
+			}
+			calledFatal = true
+			require.Equal(t, 1, code, "Unexpected error code")
+			require.Equal(t, expected, e)
+		})
+		testSnapshotsCommon(t, newDeleteCommand, cmdArgs, &snapshots, expected, true)
+	}()
+}
```

Comment:
Need to assert that fatal callback was called
`require.True(t, calledFatal, msg)`

Same in other tests where this is being done.

Reply:
Fixed: https://github.com/libopenstorage/stork/pull/191/files#diff-f2f14386f17b4772c682fe79cfd98da5R69